OasisDEX/oasis-borrow
# 🧿 Ajna position loading

# 🧿 Ajna position loading

So far without displaying its data on the UI.
  
## Changes 👷‍♀️

- Created observable that loads position metadata based on it's ID from url,
- utilized method from library for loading position data,
- added handling for owner and non owners view in manage position - banner and hiding primary button (`isOwner` is set to `true` if user is on open flow since there is no real owner of that page),
- added redirect to `404` when there is no Ajna position of given ID,
  
## How to test 🧪

- Go to `/ajna/position/1?network=goerli`,
- see if page is loading and test its interface from three context: owner, non owner and disconnected wallet.
